src/materials/ConcreteThermalMoisture.C

```diff
@@ -27,57 +27,50 @@ ConcreteThermalMoisture::validParams()
   params.addRequiredParam<std::string>(
       "type", "A string representing the Moose Object that is used to call this class");
 
-  // parameters for thermal properties calculations
-  MooseEnum thermal_conductivity_model("CONSTANT ASCE-1992 KODUR-2004 EUROCODE-2004", "CONSTANT");
-  MooseEnum thermal_capacity_model("CONSTANT ASCE-1992 KODUR-2004 EUROCODE-2004", "CONSTANT");
-  MooseEnum aggregate_type("Siliceous Carbonate", "Siliceous");
-
+  // available model for thermal and moisture transport
+  MooseEnum thermal_transport_model("CONSTANT ASCE-1992 KODUR-2004 EUROCODE-2004", "CONSTANT");
   params.addParam<MooseEnum>(
-      "thermal_conductivity_model", thermal_conductivity_model, "thermal conductivity models");
+      "thermal_transport_model", thermal_transport_model, "thermal conductivity models");
+  MooseEnum moisture_transport_model("Bazant Mensi Xi", "Bazant");
   params.addParam<MooseEnum>(
-      "thermal_capacity_model", thermal_capacity_model, "thermal capacity models");
-  params.addParam<MooseEnum>("aggregate_type", aggregate_type, "siliceous or carbonate");
-
-  params.addParam<Real>(
-      "ref_density_of_concrete", 2231.0, "refernece density of porous media Kg/m^3");
-  params.addParam<Real>(
-      "ref_specific_heat_of_concrete", 1100.0, "reference specific heat of concrete J/Kg/0C");
-  params.addParam<Real>("ref_thermal_conductivity_of_concrete",
-                        3.0,
-                        "reference thermal conductivity of concrete  W/m/0C");
+      "moisture_transport_model", moisture_transport_model, "moisture diffusivity models");
 
-  // parameters for mositure capacity (dW/dH) and humidity diffusivity calculations
+  // concrete properties
   MooseEnum cement_type("1 2 3 4", "1");
+  MooseEnum aggregate_type("Siliceous Carbonate", "Siliceous");
   MooseEnum aggregate_pore_type("dense porous", "dense");
   params.addParam<MooseEnum>(
       "cement_type", cement_type, "cement type input for moisture capacity calculations");
+  params.addParam<MooseEnum>("aggregate_type", aggregate_type, "siliceous or carbonate");
```

Comment:
The description should be changed from "siliceous or carbonate" to simply "Type of aggregate" (which admittedly doesn't say much more than the command name). The documentation system will show what the options in that enum are, so you don't need to say that in the description.